extensionsGallery-insider.json

```diff
@@ -3822,15 +3822,72 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+				{
+					"extensionId": "78",
```

Comment:
Another extension was just added with id 78, please update to 79.

Reply:
Hi @Charles-Gagnon 
I did the changes (f0b5a1f) 
Thanks